src/main/java/SpringAIController.java

```diff
@@ -0,0 +1,45 @@
+import com.openai.services.AIServiceHelper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * @author madhankumar
+ */
+@RestController
+@RequestMapping("/api/v1")
```

Comment:
Annotation 'RestController' should be placed before 'RequestMapping' for consistency.